pkg/platform/packet/packet_test.go

```diff
@@ -355,3 +355,86 @@ func TestTerraformAddDeps(t *testing.T) {
 		})
 	}
 }
+
+func Test_findRepeatingString(t *testing.T) {
+	tests := []struct {
+		name string
+		strs []string
+		want string
+	}{
+		{
+			name: "repeated_strings",
+			strs: []string{"1", "2", "3", "1"},
+			want: "1",
+		},
+		{
+			name: "unique_strings",
+			strs: []string{"1", "2", "3"},
+			want: "",
+		},
+		{
+			name: "empty_input",
+			strs: []string{},
+			want: "",
+		},
+	}
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			if got := findRepeatingString(tt.strs); got != tt.want {
+				t.Errorf("findRepeatingString() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func Test_validateNodePrivateCIDRs(t *testing.T) {
```

Comment:
All those tests can be added to `TestCheckValidConfig` array.

Reply:
I think testing things at the function level using unit test is a good strategy to write readable code.